### PR TITLE
Fixed MessageContentCell's subviews frame when size equal to .zero

### PR DESCRIPTION
**Pull request by zhongwuzw**

### CODE REVIEW

File: Sources/Views/Cells/MessageContentCell.swift [hunk 1]
```diff
@@ -171,7 +171,10 @@ open class MessageContentCell: MessageCollectionViewCell {
     /// Positions the cell's `AvatarView`.
     /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
     open func layoutAvatarView(with attributes: MessagesCollectionViewLayoutAttributes) {
-        guard attributes.avatarSize != .zero else { return }
+        guard attributes.avatarSize != .zero else {
```
Comment (SD10) [inline Sources/Views/Cells/MessageContentCell.swift]: Actually, we want to just remove these lines completely. We still need to calculate the `X` and `Y` positions

Comment (SD10) [reply Sources/Views/Cells/MessageContentCell.swift]: No early exit

Comment (zhongwuzw) [reply Sources/Views/Cells/MessageContentCell.swift]: 🤔 What's the reason? I think each subview did not depend on other's `frame`.

Comment (SD10) [reply Sources/Views/Cells/MessageContentCell.swift]: @zhongwuzw they do, I lay views out relative to each other, so we have to set X and Y

Comment (zhongwuzw) [reply Sources/Views/Cells/MessageContentCell.swift]: Oops, I didn't look carefully.

**Pull request merged**